src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/SentReferralSummaryForDashboardDTO.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ReferralForDashboard
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class SentReferralSummaryForDashboardDTO(
+  val id: UUID,
+  val sentAt: OffsetDateTime,
+  val sentBy: AuthUserDTO,
+  val referenceNumber: String,
+  val assignedTo: AuthUserDTO?,
+  val serviceUser: ServiceUserDTO?,
+  val serviceProvider: ServiceProviderDTO?,
+  val supplementaryRiskId: UUID,
```

Comment:
why does this class need the risk Id?

Reply:
Might have come in by mistake, I will remove it